cloudflare/resource_cloudflare_page_rule_test.go

```diff
@@ -438,6 +438,27 @@ func TestAccCloudflarePageRuleCacheKeyFields2(t *testing.T) {
 	})
 }
 
+func TestAccCloudflarePageRuleCacheTTLByStatus(t *testing.T) {
+	var pageRule cloudflare.PageRule
+	domain := os.Getenv("CLOUDFLARE_DOMAIN")
+	zoneID := os.Getenv("CLOUDFLARE_ZONE_ID")
+	target := fmt.Sprintf("test-cache-ttl-by-status.%s", domain)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckCloudflarePageRuleDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckCloudflarePageRuleConfigCacheTTLByStatus(zoneID, target),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckCloudflarePageRuleExists("cloudflare_page_rule.test", &pageRule),
```

Comment:
Instead of hardcoding this to "test" which may collide with other resources, could we use a random string and pass it into the configuration instead?

Similar to https://github.com/terraform-providers/terraform-provider-cloudflare/blob/639c5c8404f2cc2ef12fe44d9510156fba1036dc/cloudflare/resource_cloudflare_access_group_test.go#L44-L68

Reply:
fixed this in 04fa7ca8